ee/codegen/src/generators/workflow-sandbox-file.ts

```diff
@@ -173,6 +194,67 @@ if __name__ == "__main__":
     });
   }
 
+  private getMockNodeExecution(
+    mock: WorkflowSandboxDatasetRowMock
+  ): python.ClassInstantiation {
+    const nodeContext = this.workflowContext.findNodeContext(mock.node_id);
+
+    if (!nodeContext) {
+      throw new Error(`Node context not found for node_id: ${mock.node_id}`);
+    }
+
+    const arguments_: python.MethodArgument[] = [];
+
+    // Generate when_condition using WorkflowValueDescriptor
+    if (!isNil(mock.when_condition)) {
+      const whenConditionAst = new WorkflowValueDescriptor({
+        workflowValueDescriptor: mock.when_condition,
+        workflowContext: this.workflowContext,
+      });
+
+      arguments_.push(
+        python.methodArgument({
+          name: "when_condition",
+          value: whenConditionAst,
+        })
+      );
+    }
+
+    // Generate then_outputs by instantiating the node's Outputs class
+    const outputsArguments: python.MethodArgument[] = Object.entries(
+      mock.then_outputs
+    ).map(([key, value]) =>
+      python.methodArgument({
+        name: key,
+        value: python.TypeInstantiation.str(String(value)),
+      })
+    );
+
+    const thenOutputsInstance = python.instantiateClass({
+      classReference: python.reference({
+        name: nodeContext.nodeClassName,
+        modulePath: nodeContext.nodeModulePath,
+        attribute: ["Outputs"],
+      }),
+      arguments_: outputsArguments,
+    });
+
+    arguments_.push(
+      python.methodArgument({
+        name: "then_outputs",
+        value: thenOutputsInstance,
+      })
+    );
+
+    return python.instantiateClass({
+      classReference: python.reference({
+        name: "MockNodeExecution",
+        modulePath: ["vellum", "workflows", "nodes", "mocks"],
```

Comment:
in the `constants.ts` file add a `VELLUM_WORKFLOW_ROOT_MODULE_PATH = ["vellum", "workflows"]` and use that constant here instead